src/linter/root.go

```diff
@@ -666,6 +666,11 @@ func (d *RootWalker) handleFuncStmts(params []meta.FuncParam, uses, stmts []node
 
 	switch {
 	case b.bareReturn && b.returnsValue:
+
```

Comment:
Please avoid empty lines like these. They can separate logical blocks, but this one is superfluous.